### PR TITLE
feat: button inner content can be aligned with alignContent prop

### PR DESCRIPTION
**Pull request by ThingsIDoForLove**
### Motivation
We normally require feature where button content can be  aligned left, right or centered

### CODE REVIEW

**Comment by ThingsIDoForLove** [comment]
> Hey @ThingsIDoForLove, thank you for your pull request 🤗. The documentation from this branch can be viewed [here](https://8148-71323749-gh.circle-artifacts.com/0/docs/index.html). Please remember to update Typescript types if you changed API.



> To be honest if we do this, it is better we pass a `contentStyle` prop.

I agree with you. @ferrannp

File: src/components/Button.js [hunk 1]
```diff
@@ -107,6 +111,11 @@ class Button extends React.Component<Props, State> {
   static defaultProps = {
     mode: 'text',
     uppercase: true,
+    contentStyle: {
```
Comment (satya164) [inline src/components/Button.js]: Please move this to styles and use an array to merge it with passed styles: `style=[styles.content, contentStyle]`

**Comment by ThingsIDoForLove** [comment]
Hi guys, any further thoughts on it.

**Comment by satya164** [comment]
@ThingsIDoForLove I reverted the unrelated changes in your PR. can you run `yarn test -u` to update the snapshots? then we will merge

**Comment by satya164** [comment]
Merged in 53e3bae7e3891249a724f63cfea41ec549ca26dd

**Comment by RichardLindhout** [comment]
@satya164 Maybe we should always fill the Touchable inside to fill all space in the button e.g. touchableStyle:{
    width: 100%;
    flex: 1;
}

**Comment by RichardLindhout** [comment]
schermafbeelding 2019-02-27 om 10 53 51

**Comment by satya164** [comment]
@RichardLindhout you're right. please send a PR if you can :)

**Comment by RichardLindhout** [comment]
Will do ;)

**Comment by RichardLindhout** [comment]
@satya164 Oops, I pushed it inside another pull request.. I don't do pull requests that often
https://github.com/callstack/react-native-paper/pull/883

**Comment by satya164** [comment]
Please make a separate PR for that.

**Comment by RichardLindhout** [comment]
@satya164 I did now ;)

Understanding pull requests a bit more know. Excited to contribute more! This library is the first quality UI framework which I have seen in React-Native. With great understanding performance (e.g. use of native driver and smooth animation ;)

Also thank for all your work and also target the web version 👍

**Comment by RichardLindhout** [comment]
Also I created a simpler version of my input pull request with less changes: https://github.com/callstack/react-native-paper/pull/887

**Pull request closed without merging**